x/mint/types/params.go

```diff
@@ -43,11 +43,11 @@ func NewParams(
 func DefaultParams() Params {
 	return Params{
 		MintDenom:           sdk.DefaultBondDenom,
-		InflationRateChange: sdk.NewDecWithPrec(13, 2),
-		InflationMax:        sdk.NewDecWithPrec(20, 2),
-		InflationMin:        sdk.NewDecWithPrec(7, 2),
+		InflationRateChange: sdk.ZeroDec(),
+		InflationMax:        sdk.NewDecWithPrec(2, 2),
+		InflationMin:        sdk.NewDecWithPrec(2, 2),
 		GoalBonded:          sdk.NewDecWithPrec(67, 2),
-		BlocksPerYear:       uint64(60 * 60 * 8766 / 5), // assuming 5 second block times
+		BlocksPerYear:       uint64(60 * 60 * 8766 / 15), // assuming 15 second block times
```

Comment:
How is this used by the Cosmos SDK? Got a link to docs maybe?

Reply:
blocks per year is used to determine the rate of change for inflation (we currently have the max inflation rate change set to 0, so this value shouldn't be important unless we ever change it)

https://docs.cosmos.network/main/modules/mint